py-polars/tests/io/test_lazy_csv.py

```diff
@@ -12,6 +13,11 @@ def test_scan_csv() -> None:
     assert df.collect().shape == (4, 3)
 
 
+def test_scan_empty_csv() -> None:
+    with pytest.raises(Exception):
+        pl.scan_csv(Path(__file__).parent.parent / "files" / "empty.csv")
```

Comment:
Can we read from an empty in memory buffer here? Then we don't need to add an empty csv file to the repo.

Reply:
That's a good idea! However, I'm not sure how to do it using python 😞 

I've tried with `bytesarray()`n `memoryview()` and `np.empty()`, but can't make it work.

Any advice?

BTW,  pipeline is failing at `Windows test run` and I'm not sure why, it doesn't seem related to my PR:

```
cargo t --all-features --test it
   Compiling semver v1.0.6
   Compiling plotters-backend v0.3.2
   Compiling half v1.8.2
   Compiling oorandom v11.1.3
   Compiling itertools v0.10.3
   Compiling textwrap v0.11.0
   Compiling plotters-svg v0.3.1
   Compiling clap v2.34.0
   Compiling winapi-util v0.1.5
   Compiling atty v0.2.14
   Compiling rustc_version v0.4.0
   Compiling plotters v0.3.1
   Compiling same-file v1.0.6
   Compiling cast v0.2.7
   Compiling walkdir v2.3.2
   Compiling serde_cbor v0.11.2
   Compiling criterion-plot v0.4.4
   Compiling polars-utils v0.20.0 (D:\a\polars\polars\polars\polars-utils)
   Compiling tinytemplate v1.2.1
   Compiling criterion v0.3.5
   Compiling polars-arrow v0.20.0 (D:\a\polars\polars\polars\polars-arrow)
   Compiling polars-core v0.20.0 (D:\a\polars\polars\polars\polars-core)
   Compiling polars-time v0.20.0 (D:\a\polars\polars\polars\polars-time)
   Compiling polars-io v0.20.0 (D:\a\polars\polars\polars\polars-io)
   Compiling polars-lazy v0.20.0 (D:\a\polars\polars\polars\polars-lazy)
LLVM ERROR: IO failure on output stream: no space on device
error: could not compile `polars-lazy`
warning: build failed, waiting for other jobs to finish...
LLVM ERROR: IO failure on output stream: no space on device
error: build failed
mingw32-make: *** [Makefile:42: integration-tests] Error 101
Error: Process completed with exit code 1.
```

Reply:
You can use `io.BytesIO` for that. That gives you an empty buffer that mimics a file handle.

Yeah. The windows ci is not related. You can ignore that. :)

Reply:
Thanks again @ritchie46, really appreciate you took some of your time to help me :)

I did all the changes you asked for.

Could you please check if everything is ok?